src/sbuf.c

```diff
@@ -29,6 +29,7 @@
 #include <usual/safeio.h>
 #include <usual/slab.h>
 #include <usual/mbuf.h>
+#include <usual/tls/tls_internal.h>
```

Comment:
The fact that I need to include a header from libusual with "internal" in the name is a smell and a sign that maybe the logic to test whether a new TLS config structure matches an existing struct should actually live in there.

Reply:
Yeah, I think it's fine for initial review, but before merging the logic to compare a `tls_config` should be created as a PR of `libusual` instead (which can then be used in this PR).

Reply:
d6ff3a09a0ae832cdf1c643f39ee626517297059

PRed here https://github.com/libusual/libusual/pull/57